crates/hyperswitch_connectors/src/connectors/aci/transformers.rs

```diff
@@ -713,12 +730,20 @@ impl TryFrom<(&AciRouterData<&PaymentsAuthorizeRouterData>, &Card)> for AciPayme
         let txn_details = get_transaction_details(item)?;
         let payment_method = PaymentDetails::try_from((card_data.clone(), card_holder_name))?;
         let instruction = get_instruction_details(item);
+        let recurring_type = get_recurring_type(item);
+        let three_ds_two_enrolled = if item.router_data.is_three_ds() {
+            Some(item.router_data.request.enrolled_for_3ds)
+        } else {
+            None
+        };
```

Comment:
Why is this change needed?
```suggestion
let three_ds_two_enrolled = item.router_data.is_three_ds().then_some(item.router_data.request.enrolled_for_3ds);
```

Reply:
Added this field so that three_ds can be performed for the cards which are not already enrolled in their simulator
https://docs.aciworldwide.com/tutorials/threeDSecure/TestingGuide